src/components/notifications/Webpush.vue

```diff
@@ -0,0 +1,106 @@
+<template>
```

Comment:
The general functionality here is there.
Please consider what a new user might see and include a bit more "fluff"-helptexts what this functionality is and how it works. (1-2 sentences)